style.css

```diff
@@ -104,7 +104,7 @@
   }
 
   main {
-    background-color: #e5e5e5;
+    background-color: #fff6f6b7;
```

Comment:
- [ ] Your background color is different from the one on the Figma design(`#f4f5f7`). Kindly check it to follow the Figma design.